src/main/java/net/wurstclient/commands/DistCmd.java

```diff
@@ -0,0 +1,94 @@
+package net.wurstclient.commands;
+
+import net.minecraft.util.math.BlockPos;
+import net.wurstclient.command.CmdException;
+import net.wurstclient.util.ChatUtils;
+import net.wurstclient.command.CmdError;
+import net.wurstclient.command.CmdException;
+import net.wurstclient.command.CmdSyntaxError;
+import net.wurstclient.command.Command;
+import java.lang.Math;
+
+
+public class DistCmd extends Command {
+
+
+    public DistCmd() {
+        super("dist", "Gets Distance between 2 Cordinates\nYou can convert Overworld Cords To Nether cords With cvt:<x> <z>\n ", ".dist <x> <z>:<x> <z>", ".dist cvt:<x> <z>");
+    }
+
+    @Override
+    public void call(String[] args) throws CmdException {
+
+        if(args.length < 1)
+            throw new CmdError("Enter Command Properly");
+
+        String Allcords = String.join(" ", args);

```

Comment:
This join doesn't accomplish much, and just adds more complexity and computational overhead